### DISCUSSION TITLE
How to create a question-dialogue with predefined answers?

### DISCUSSION THREAD

**Discussion by Gewerd-Strauss** [SilentVoid13/Templater#734] [Help] [answered]
Hello,

I am currently setting up quite a lot of templates, and I just now realising I don't know how to do so.
As an example, I want a simple Yes-No-Modal for the question "Add matriculation number?". Choosing "yes" will set the Constant `MatrNum` to its correct Value, f.e. `12345`. Choosing "no" results in an empty string which is therefore not show in the resulting output.

I am unsure as to how I would do so, and would appreciate help.

Sincerely,
~Gw

**Comment by AB1908** [comment, marked as answer]
Look at `tp.system.suggester` in the docs.

**Reply by Gewerd-Strauss** [reply]
I was apparently really damn blind when looking through the docs yesterday ._. Thank you.

**Reply by AB1908** [reply]
Happens to all of us 😄
